src/components/search.tsx

```diff
@@ -323,7 +323,3 @@ const getEntity = (e: ResultType) => {
       "operator";
   }
 };
-
-const removeDiacritics = (str: string) => {
-  return str.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
-};
```

Comment:
So it's not needed to remove diacritics anymore, or is it handled in some other way?

Reply:
It is handled by the Autocomplete component, see https://mui.com/material-ui/react-autocomplete/#custom-filter 

> config.ignoreAccents (bool [optional]): Defaults to true. Remove diacritics.

Reply:
Cool, thanks for explaining 👍